src/extract.ts

```diff
@@ -133,6 +133,39 @@ export interface PytestBenchmarkJson {
     version: string;
 }
```

Comment:
I think it would be helpful to add a union type for the estimate, group, and unsupported types
```suggestion
}

type JuliaBenchmark = JuliaBenchmarkGroup | JuliaBenchmarkTrialEstimate | JuliaBenchmarkOther

type JuliaBenchmarkOther = [string, unknown]

```